src/main/java/com/stripe/model/Invoice.java

```diff
@@ -25,12 +26,14 @@
 	Long date;
 	String description;
 	Discount discount;
+	Long dueDate;
```

Comment:
The API docs also list `days_until_due` as a create parameter. Is there a reason we're only exposing  the due date parameter?